src/NLog.Targets.Seq/LogEventInfoFormatter.cs

```diff
@@ -91,7 +93,7 @@ public static void ToCompactJson(LogEventInfo logEvent, TextWriter output, IList
                     var space = new StringWriter();
                     var formatString = "{0:" + r.Format + "}";
 
-                    if (!logEvent.MessageTemplateParameters.IsPositional && logEvent.Properties != null && logEvent.Properties.ContainsKey(r.Name))
+                    if (!mtp.IsPositional && logEvent.Properties != null && logEvent.Properties.ContainsKey(r.Name))
```

Comment:
@snakefoot thanks for the feedback; there's a check here on this line, I'm not sure where the other check you're referring to could be, would you mind pointing me to the line in question? Many thanks!

Reply:
@nblumhardt It is line 145/146